packages/components/buttons/secondary-button/src/secondary-button.styles.ts

```diff
@@ -31,19 +32,14 @@ const getStateStyles = (
   if (isActive) {
     const baseActiveStyles = [
       css`
-        box-shadow: ${designTokens.shadow9};
+        border: ${designTokens.borderForButtonAsSecondaryWhenActive};
+        box-shadow: ${designTokens.shadowForButtonWhenActive};
         background-color: ${designTokens.colorSurface};
         &:focus,
         &:hover {
           background-color: ${designTokens.colorNeutral95};
         }
       `,
-      isDisabled &&
```

Comment:
This styles were never used. If the `isDisabled` prop is true, we return other styles previously in this function.